src/framework/core/js/ModelTransformationTransforms.js

```diff
@@ -740,4 +740,159 @@ var fluid = fluid || fluid_2_0_0;
             (transformSpec.max === undefined ||  transformSpec.max >= value) ? true : false;
     };
 
+    /**
+     *
+     * Convert a string to a Boolean, for example, when working with HTML form element values.
+     *
+     * The following are all false: undefined, null, "", "0", "false", false, 0
+     *
+     * Everything else is true.
+     *
+     * @param value {String} The value to be interpreted.
+     * @returns {Boolean} The interpreted value.
+     */
+    fluid.transforms.stringToBoolean = function (value) {
+        if (value) {
+            return !Boolean(value === "" || value === "0" || value === "false");
+        }
+        else {
+            return false;
+        }
+    };
+
+    fluid.defaults("fluid.transforms.stringToBoolean", {
```

Comment:
You should register these two transforms as inverses of each other, and supply tests that the inversion is (sufficiently) faithful, as with the other ModelTransformationTests later in the file